v1/types/types.gunk

```diff
@@ -271,30 +271,39 @@ type Profile struct {
 	// ProfileID is the unique identifier of a profile.
 	ProfileID string `pb:"1" json:"profile_id"`
 
-	// Account name
-	Name string `pb:"2" json:"name"`
+	// Full name
+	FullName string `pb:"2" json:"fullname"`
+
+	// User name
+	UserName string `pb:"3" json:"username"`
+
+	// Birth date
+	BirthDate string `pb:"4" json:"birthdate"`
 
 	// Language code used
-	Language string `pb:"3" json:"language"`
+	Language string `pb:"5" json:"language"`
```

Comment:
side note, try not to change the pb field number once it's in use

Reply:
It's not in use yet.